### PR TITLE
Update beta visual select

### PR DESCRIPTION
**Pull request by sgrg1**
selected item on a combo box was not working when returning to previous page .

### CODE REVIEW

File: src/components/animalBreedType.jsx [hunk 1]
```diff
@@ -14,6 +14,7 @@ const AnimalBreedType = ({
   animalSubCategories,
   shouldShow,
 }) => {
+  const animalBreedType= rest.formik.values.animalBreedType;
```
Comment (martypowell) [inline src/components/animalBreedType.jsx:17]: there is an easier way to do this if your variable name is the same as the last property

```js
const { animalBreedType } = rest.formik.values;
```

See

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring

File: src/components/animalColorType.jsx [hunk 2]
```diff
@@ -15,6 +15,7 @@ const AnimalColorType = ({
   AnimalColors,
   shouldShow,
 }) => {
+const  animalColorType = rest.formik.values.animalColorType;
```
Comment (martypowell) [inline src/components/animalColorType.jsx:18]: same comment applies here, https://github.com/baltimorecounty/react-baltcogo/pull/205/files#r418641857

File: src/components/otherAnimalTypes.jsx [hunk 3]
```diff
@@ -12,6 +12,7 @@ const OtherAnimalTypes = ({
   OtherAnimalTypes,
   shouldShow,
 }) => {
+  const otherAnimalTypes  =rest.formik.values.otherAnimalTypes;
```
Comment (martypowell) [inline src/components/otherAnimalTypes.jsx:15]: same comment applies here, https://github.com/baltimorecounty/react-baltcogo/pull/205/files#r418641857

File: src/components/petType.jsx [hunk 4]
```diff
@@ -11,6 +11,7 @@ const PetType = ({
   PetTypes,
   shouldShow,
 }) => {
+  const petTypeID = rest.formik.values.petTypeID;
```
Comment (martypowell) [inline src/components/petType.jsx:14]: same comment applies here, https://github.com/baltimorecounty/react-baltcogo/pull/205/files#r418641857

File: src/components/requestCategory.jsx [hunk 5]
```diff
@@ -3,18 +3,20 @@ import ErrorMsg from "./ErrorMessage";
...
 const RequestCategory = ({
+  requestType,
 	errorsRequestType,
 	touchedRequestType,
 	pageFieldName,
 	handleServiceRequestChange,
 	rest,
 	Categories
 }) => {
+  const requestTypeID = rest.formik.values.requestTypeID;
```
Comment (martypowell) [inline src/components/requestCategory.jsx:14]: same comment applies here, https://github.com/baltimorecounty/react-baltcogo/pull/205/files#r418641857

File: src/components/requestSubCategory.jsx [hunk 6]
```diff
@@ -10,6 +10,7 @@ const SubCategory = ({
   rest,
   subCategories
 }) => {
+  const subRequestTypeID = rest.formik.values.subRequestTypeID;
```
Comment (martypowell) [inline src/components/requestSubCategory.jsx:13]: same comments applies here, https://github.com/baltimorecounty/react-baltcogo/pull/205/files#r418641857

File: src/components/sexType.jsx [hunk 7]
```diff
@@ -12,6 +12,7 @@ const SexType = ({
   animalSex,
   shouldShow,
 }) => {
+  const sexType = rest.formik.values.sexType;
```
Comment (martypowell) [inline src/components/sexType.jsx:15]: same comments applies here, https://github.com/baltimorecounty/react-baltcogo/pull/205/files#r418641857

File: src/components/animalBreedType.jsx (hunk 1 above)
Comment (sgrg1) [reply src/components/animalBreedType.jsx]: done

File: src/components/animalColorType.jsx (hunk 2 above)
Comment (sgrg1) [reply src/components/animalColorType.jsx]: done

File: src/components/otherAnimalTypes.jsx (hunk 3 above)
Comment (sgrg1) [reply src/components/otherAnimalTypes.jsx]: done

File: src/components/petType.jsx (hunk 4 above)
Comment (sgrg1) [reply src/components/petType.jsx]: done

File: src/components/requestCategory.jsx (hunk 5 above)
Comment (sgrg1) [reply src/components/requestCategory.jsx]: done

File: src/components/sexType.jsx (hunk 7 above)
Comment (sgrg1) [reply src/components/sexType.jsx]: done

File: src/components/otherAnimalTypes.jsx [hunk 8]
```diff
@@ -12,6 +12,7 @@ const OtherAnimalTypes = ({
   OtherAnimalTypes,
   shouldShow,
 }) => {
+  const {otherAnimalTypes}  =rest.formik.values;
```
Comment (martypowell) [inline src/components/otherAnimalTypes.jsx:15]: do you have prettier installed? it doesn't look like any of these files are being formatted?

Comment (sgrg1) [reply src/components/otherAnimalTypes.jsx]: yes i do, may be i forgot to save it

**Pull request merged**